cmake/std/atdm/utils/get_known_system_info.sh

```diff
@@ -66,6 +66,9 @@ elif [[ $ATDM_CONFIG_REAL_HOSTNAME == "ghost"* ]] \
   || [[ $ATDM_CONFIG_REAL_HOSTNAME =~ gho[0-9]+ ]] ; then
   ATDM_HOSTNAME=ghost
   ATDM_SYSTEM_NAME=serrano
+elif [[ $ATDM_CONFIG_REAL_HOSTNAME == "doom"* ]] ; then
```

Comment:
So this would change from 'doom' to 'cts1'.  (Let's see if we can get that to work.)